controllers/articles-controllers

```diff
@@ -0,0 +1,13 @@
+const { fetchArticleById } = require("../models/articles-models");
+
+function getArticleById(request, response, next) {
+  const { article_id } = request.params;
+
+  fetchArticleById(article_id)
+    .then((article) => {
+      response.status(200).send({ article });
+    })
+    .catch((error) => next(error));
+}
+
```

Comment:
Lovely work sticking to naming conventions here, it makes your code really easy to navigate 😄